runtime/flang/CMakeLists.txt

```diff
@@ -477,16 +474,25 @@ SET(FTN_SUPPORT
   xfer_rpm1.c
   )
 
+add_library(iso_c_bind
```

Comment:
These libraries don't have none of the compile definitions the others have. Also we need 2 variants. One with `-DPG_DLL` and one without.

Reply:
I completely disabled this functionality in gh-93 after I realized my mistake. These libraries serve no purpose now but to fix the dependency issues.

Reply:
But, `ieee_arithmetic` will now use a `.mod` file generated from this library right?

Reply:
The dependencies are specified with `add_dependencies` rather than `OBJECT_DEPENDS` which doesn't work with the Ninja generator and Fortran. I filed a CMake issue linked above.